launch/config.ts

```diff
@@ -0,0 +1,73 @@
+import BN from 'bn.js';
+
+const config = {
```

Comment:
I guess it's better to use json file for other languages.

Reply:
I'll test if string literal works, at the beginning I changed it to bignumber for testing if it's that causing errors